README.md

```diff
@@ -21,3 +21,28 @@ If your project is interested in joining the Node.js Foundation please read the
 Project mentorship is not a technical role. In fact, mentors are discouraged from giving technical advice to projects. Instead, the purpose of mentorship is to encourage and improve a projects ability to be participatory, transparent, and effective. Mentors are there to help projects adopt and iterate on policies and processes that achieve these goals and eventually allow them to graduate the incubation phase.
 
 * Mikeal Rogers (@mikeal)
+
+## Policy Change Proposal Process
+
+The Node.js TSC is chartered to oversee the technical governance of all Top
+Level Projects and Working Groups under the Node.js Foundation. The TSC
+establishes the default governance, conduct, and licensing policies for all Top
+Level Projects. Top Level Projects and Working Groups have broad powers of
+self-governance.
+
+To propose a change or addition to policies or prcesses that are intended to
+cover all Top Level Projects and Working Groups in the foundation, a PR should
+be opened in the `nodejs/TSC` repository. 
+
+The pull request can be labeled `tsc-agenda` to request that it be put on the
+agenda for the next TSC meeting.
+
+The Node.js Foundation Board of Directors retains certain rights (especially
+legal considerations). If the TSC endorses a proposal, they will escalate to the
```

Comment:
Perhaps, `If the TSC endorses a change proposal for any policy or guideline that falls under the responsibility of the Node.js Foundation Board, the TSC will escalate...`